### PR TITLE
break_me.py

### PR DESCRIPTION
**Pull request by alexLaws**
Lesson 1 break_me.py

### CODE REVIEW

File: students/alexLaws/lesson02/grid_printer.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+def grid_printer():
```
Comment (natasha-aleksandrova) [inline students/alexLaws/lesson02/grid_printer.py]: Alex, you'll want your edit set to exactly 4 spaces (not tabs) for indentation

Comment (natasha-aleksandrova) [reply students/alexLaws/lesson02/grid_printer.py]: https://www.python.org/dev/peps/pep-0008/#tabs-or-spaces

**Comment by natasha-aleksandrova** [comment]
@alexLaws please make sure your editor is set to 4 spaces for indentation.

**Comment by natasha-aleksandrova** [comment]
@alexLaws looks like this PR also include grid printer but I don't see submission in EdX

can you please go and submit in EdX so you can get the grade?

**Comment by alexLaws** [comment]
Hey Natasha,

I didn't mean to include this in the pull request.  I submitted the pull request just through break_me.py but then I guess it added the grid printer when I committed it.  I have some more work to do to code parts 2/3/etc

**Comment by natasha-aleksandrova** [comment]
no worries, just do another PR when you're ready 🙂 

instead of `git add .` you can just do `git add <filename>` if you're not ready yet to add all of your work to git

**Pull request merged**